circe-akka-serializer/src/main/scala/org/virtuslab/ash/circe/CirceAkkaSerializer.scala

```diff
@@ -119,22 +124,26 @@ abstract class CirceAkkaSerializer[Ser <: AnyRef: ClassTag](system: ExtendedActo
     val foundSerializables = reflections.getSubTypesOf(classTag[Ser].runtimeClass).asScala.filterNot(_.isInterface)
     foundSerializables.foreach { clazz =>
       try {
-        val codec = codecsMap(parents(clazz.getName))
-        assert(codec._1 != null)
-        assert(codec._2 != null)
+        codecsMap(parents(clazz.getName))
       } catch {
         case e @ (_: NoSuchElementException | _: NotSerializableException) =>
```

Comment:
Hmmm how can a `NotSerializableException` be thrown from two `Map.apply` calls in the `try` block? (AFAICS)

Reply:
Also, is that exception even relevant here? it seems to correspond to Java serialization, right?

Reply:
> Hmmm how can a `NotSerializableException` be thrown from two `Map.apply` calls in the `try` block? (AFAICS)

Yeah, you are right, it can't

Reply:
> Also, is that exception even relevant here? it seems to correspond to Java serialization, right?

This exception is very relevant for Akka serializers. From documentation:

>NotSerializableException is treated as a transient problem in the TCP based remoting layer. The problem will be logged and message is dropped. Other exceptions will tear down the TCP connection because it can be an indication of corrupt bytes from the underlying transport.